### PR TITLE
add imagecredentials to triton docker image

### PR DESCRIPTION
**Pull request by robinvd**
**What this PR does / why we need it**:

inference and statistics have imageCredentials support, only triton image is missing this functionality

**Checklist**
- [x] Reviewed the [`CONTRIBUTING.md`](https://github.com/allegroai/clearml-helm-charts/blob/main/CONTRIBUTING.md#pull-requests) guide (**required**)
- [x] Verify the work you plan to merge addresses an existing [issue](https://github.com/allegroai/clearml-helm-charts/issues) (If not, open a new one) (**required**)
- [x] Check your branch with `helm lint` (**required**)
- [x] Update `version` in `Chart.yaml` according [semver](https://semver.org/) rules (**required**)
- [x] Substitute `annotations` section in `Chart.yaml` annotating implementations (useful for Artifecthub changelog) (**required**)
- [x] Update chart README using [helm-docs](https://github.com/norwoodj/helm-docs) (**required**)

**Which issue(s) this PR fixes**:

Fixes https://github.com/allegroai/clearml-helm-charts/issues/243

**Special notes for your reviewer**:

### CODE REVIEW

File: charts/clearml-serving/templates/clearml-serving-triton-deployment.yaml [hunk 1]
```diff
@@ -19,6 +19,14 @@ spec:
         clearml.serving.network/clearml-serving-backend: "true"
         clearml.serving.service: {{ include "clearmlServing.fullname" . }}-triton
     spec:
+      {{- if .Values.imageCredentials.enabled }}
```
Comment (janwytze) [inline charts/clearml-serving/templates/clearml-serving-triton-deployment.yaml:22]: Shouldn't `imageCredentials` exist in the `values.yaml` like the other charts?

https://github.com/allegroai/clearml-helm-charts/blob/c8aaf91f52a750e95930a9aa6e1264a1d1c3b4cd/charts/clearml/values.yaml#L7

**Review by janwytze — CHANGES_REQUESTED** [review]

File: charts/clearml-serving/templates/clearml-serving-triton-deployment.yaml (hunk 1 above)
Comment (robinvd) [reply charts/clearml-serving/templates/clearml-serving-triton-deployment.yaml:22]: fixed

**Comment by valeriano-manassero** [comment]
pls complete two missing points from PR template so I can proceed to review.

**Comment by robinvd** [comment]
@valeriano-manassero added the missing points

**Comment by valeriano-manassero** [comment]
> @valeriano-manassero added the missing points

ty, ci passed without problems, I just have a small request, from `Chart.yaml` pls remove:

```
    - kind: changed
      description: removed deprecated networkPolicy
    - kind: fixed
      description: missng ingresses className
```

these are part of previous version and artifact hub just wants implementations on specific version. once done I can restart CI and merge. ty.

**Pull request merged**